app/policies/linked_profile_policy.rb

```diff
@@ -0,0 +1,11 @@
+class LinkedProfilePolicy < ApplicationPolicy
+  def show?
+    record.user == user || user.has_role?(:admin)
```

Comment:
Should probably be `record.public? || record.user == user || is_admin?` because:

1. linked profiles need to be public so we can show 'em on your profile
2. `is_admin?` is a shorthand for `user.has_role?(:admin)`